### ISSUE TITLE
Crash in the owner stack UI

### ISSUE THREAD

**Issue by gaearon** [bvaughn/react-devtools-experimental#97]
1. "Enter" into HOC-31
2. "Enter" into HOC-30 inside of it

```
Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at devtools.js:16187
    at commitHookEffectList (devtools.js:6110)
    at commitLifeCycles (devtools.js:6110)
    at commitAllLifeCycles (devtools.js:6323)
    at HTMLUnknownElement.callCallback (devtools.js:2943)
    at Object.invokeGuardedCallbackDev (devtools.js:2963)
    at invokeGuardedCallback (devtools.js:2979)
    at commitRoot (devtools.js:6355)
    at devtools.js:6667
    at Object.unstable_runWithPriority (devtools.js:11970
```

I think it crashes because `ownerStack.length` has changed but we're _already_ hiding them so there's actually just two children.

We need to make it so that change in owner stack size forces it to _not_ overflow at first, so we can measure it again.

**Comment by bvaughn** [comment]
Thanks for the repro. I'll look into it today.

**Comment by bvaughn** [comment]
Fixed with e43cfa0
